jobserver/fields.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.utils.module_loading import import_string
+
+
+def parse_roles(paths):
+    """Convert Role dotted paths to Role objects"""
+    return [import_string(p) for p in paths]
```

Comment:
I wonder if some basic validation of the import path is worth doing here, given this is effectively arbitrary code execution?

E.g. check `p.startswith('jobserver.authorization.roles')`

Or even just have the base class name in `roles` field, and add (and thus enforce) `jobserver.authorization.roles` prefix here?

Reply:
Fixed and rebased in with a check at both the save and read sites.